cloudconfig/instancecfg/instancecfg.go

```diff
@@ -300,7 +300,7 @@ type StateInitializationParams struct {
 	CustomImageMetadata []*imagemetadata.ImageMetadata
 }
 
-type stateInitializationParamsInternal struct {
+type StateInitializationParamsInternal struct {
```

Comment:
If this is now exported. needs comment.
Perhaps need to rename since by being exported it's no longer "internal"